src/Uno.UI/UI/Xaml/Controls/Flyout/Flyout.cs

```diff
@@ -22,8 +22,8 @@ namespace Windows.UI.Xaml.Controls
 	public partial class Flyout : FlyoutBase
 	{
 		private readonly SerialDisposable _sizeChangedDisposable = new SerialDisposable();
-		internal protected readonly FlyoutPresenter _presenter = new FlyoutPresenter();
-		internal protected Popup _popup;
+		protected internal readonly FlyoutPresenter _presenter = new FlyoutPresenter();
```

Comment:
This will become protected **or** internal. Is that the intent ?

Reply:
Actually this is just a cosmetic change, but IMO this should become 'private protected' (probably the original intention).